source/customization.rst

```diff
@@ -1,4 +1,4 @@
-.. _customization:
+...................... _customization:
```

Comment:
Is this some typo? It shows up as the literal `…………………. _customization:`

Reply:
Fixed the typo. It no longer appears in the diff.